### PR TITLE
add negive and positive examples

### PR DESCRIPTION
**Pull request by sevenzing**

### CODE REVIEW

File: Docs/Examples.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+```
+
+```java
+fibs: List<Integer> = []
+fibs.Append("2") // <- error: List<Integer>.Append 
+                 // expects first agrument of type Integer
+```
+
+```java
+lst = [1, False] // <-- error: cannot construct lists of different type
```
Comment (DOCtorActoAntohich) [inline Docs/Examples.md]: `cannot construct lists of different type` -> `cannot construct a list from elements of different types`

File: Docs/Examples.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    a = [1] // <-- error: cannot assign type List<Integer> to type Integer 
+end
+```
+
+```java
+method number(): Integer is
+    return 42
+end
+
+IO().Write(number()) // error: IO().Write expected String as first argument, found Integer
```
Comment (DOCtorActoAntohich) [inline Docs/Examples.md]: `this.number()`

File: Docs/Examples.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+
+```java
+while 1 loop // error: While expected type Boolean, found Integer
+end
+```
+
+```java
+class Tuple<A, B> is
+    this (first: A, second: B) is
+        this.first = first
```
Comment (DOCtorActoAntohich) [inline Docs/Examples.md:142]: Add fields to class declaration (?)

Comment (sevenzing) [reply Docs/Examples.md:142]: like that?
```java

class Tuple<A, B> is
    first: A
    second: B
    this (first: A, second: B) is
        ...
     end
end
```

Comment (DOCtorActoAntohich) [reply Docs/Examples.md:142]: yes

Comment (DOCtorActoAntohich) [reply Docs/Examples.md:142]: but inside the class (im dum)

**Comment by ntdesmond** [comment]
Good job now add `var` everywhere 👍🏼 
*on every *declaration*

UPD:
Also consider using `ts` instead of `java` in markdown code blocks

**Review by DOCtorActoAntohich — APPROVED** [review]

**Pull request merged**